package.json

```diff
@@ -378,6 +378,7 @@
     "mime": "^2.3.1",
     "minimatch": "^3.0.2",
     "optimist": "^0.6.1",
+    "puppeteer": "^1.9.0",
```

Comment:
@johnjbarton, hi :-) Was this meant to be added under `devDependencies` instead?

Reply:
@edmorley This is already fixed on master. See https://github.com/karma-runner/karma/issues/3191.

Reply:
Ah sorry missed that - thank you :-)